app/controllers/admin_controller.rb

```diff
@@ -102,4 +104,29 @@ def culled_games
   def about
   end
 
+  def stats_display
```

Comment:
Also, why is this named `stats_display`? Why not `stats`? Since it's only a JSON endpoint, the word "display" is a bit confusing.